Documentation/Books/Manual/Administration/Configuration/README.md

```diff
@@ -13,6 +13,9 @@ General Options
 Prints a list of the most common options available and then exits. In order to
 see all options use *--help-all*.
 
+To make use of the startup options from a program, the option `--dump-options`
```

Comment:
To receive the startup options in JSON format pass the `--dump-options` flag. This will print out all options and exit.